kernel/src/main/scala/cats/kernel/PartialOrder.scala

```diff
@@ -154,10 +154,13 @@ object PartialOrder extends PartialOrderFunctions[PartialOrder] {
       def partialCompare(x: A, y: A) = f(x, y)
     }
 
-  /**
-   * Implicitly convert a `PartialOrder[A]` to a
-   * `scala.math.PartialOrdering[A]` instance.
-   */
+  def fromPartialOrdering[A](implicit ev: PartialOrdering[A]): PartialOrder[A] = from[A] { (a, b) =>
+    ev.tryCompare(a, b).fold(Double.NaN)(_.toDouble)
+  }
+}
```

Comment:
I'd prefer extending the `PartialOrder` trait here instead of having so many lambdas (`from` and the `fold`).